src/Godot.Bindings/Bridge/ClassDB/ClassDB.cs

```diff
@@ -412,7 +413,7 @@ private unsafe static void Free_Native(void* userData, void* instance)
 
         if (!context.RegisteredVirtualMethodOverrides.TryGetValue(methodNameStr, out var virtualMethodInfo))
         {
-            throw new InvalidOperationException($"Virtual method '{methodNameStr}' has not been registered in class '{context.ClassName}'.");
+            return null;
```

Comment:
It would be nice to add a comment here explaining why we're returning if we can't find it.

```suggestion
            // Virtual method not registered, it likely means it wasn't overridden.
            // Returning null so it falls back to the default implementation.
            return null;
```